### ISSUE TITLE
Server crashes due to tickspeed

### ISSUE THREAD

**Issue by TheLastLaugh** [Sunekaer/stoneBlock#761]
## Bug Report
UPDATE - Sorry about not uploading the log before. - UPDATE
It's a little hard to explain, but the server has been running fine for the last week or so, however recently it's started to slow down. It's gotten to a point where the server finishes loading, then after some time (say a minute or two) the server says that it took over a minute to do 1 tick and is treating it like a crash. I'm not too sure if I explained it well, but you'll see in the log

## Steps to Reproduce (for bugs)

1.Uhhhh.... I guess have the server and game running on a single pc
2. Play stoneblock for a while... Not sure what else

## Logs

* Client/Server Log: https://gist.github.com/TheLastLaugh/4bcb4a1f26c8adf6842a8a1c19b82417

## Client Information

* Modpack Version: 1.0.32
* Server/LAN/Single Player: Server
* Optifine Installed: False
* Shaders Enabled: False

**Comment by Teslachu** [comment]
We recently had this issue on our server after accidentally breaking a storage drawer with several upgrades of cobblestone. Like you, we had to change the max tick time to allow the server to stay running after initial boot up. We tried using the /kill @e[type=item] command to kill all the item entities but would say found no items. Since we were in the area where the cobble was laying. We managed to very slowly get the drawer and upgrades replaced in the ground and insert the cobble in slowly. As the entity numbers started going down, the "speed" of the server increased and the lag was resolved.  
TL:DR you most likely have item/entity lag, will take time at slow server tick speed to find and resolve the issue.

**Comment by TheLastLaugh** [comment]
Thanks Teslachu.
Weirdly enough, that wasn't the problem. The problem happened to be that my friend on the server was making 100,000 flux, 100,000 stone, and 100,000 growth accelerators. He completely forgot about it, and when it was finished, the server cleared up. He went to make another 100,000 flux, then it started to slow down again.
So yeah, TL:DR, auto-crafting is a pain in the backside

**Comment by Teslachu** [comment]
Was he using refined storage? I know in some packs it can REALLY cause some massive server lag, from the auto crafting. I completely forgot about that. I am glad you guys got it fixed though!

**Comment by nzminecraftDOTcom** [comment]
I had to shut down refined storage from being used on my servers, one person started mass-auto-crafting and that's all it took.
Also try /cofh killall [type] e.g item, arrow etc from console if there are a lot of items on the ground.
